src/features/student/modules/Rating.tsx

```diff
@@ -0,0 +1,33 @@
+import styles from './Rating.module.scss';
+
+export type RatingType = 'good' | 'median' | 'bad' | 'none';
+
+const RATING_CLASSNAMES: Record<RatingType, string> = {
+    good: styles.ratingGood,
+    median: styles.ratingMedian,
+    bad: styles.ratingBad,
+    none: styles.ratingNone,
+};
+
+const getRatingClass = (rating: number) => {
+    if (rating <= 0) {
+        return RATING_CLASSNAMES.none;
+    } else if (rating < 4) {
+        return RATING_CLASSNAMES.bad;
+    } else if (rating < 5) {
+        return RATING_CLASSNAMES.median;
+    } else {
+        return RATING_CLASSNAMES.good;
+    }
+};
```

Comment:
I personally think a switch would be more readable
```suggestion
const getRatingClass = (rating: number) => {
    switch (true) {
        case rating <= 0:
            return RATING_CLASSNAMES.none;
        case rating < 4:
            return RATING_CLASSNAMES.bad;
        case rating < 5:
            return RATING_CLASSNAMES.median;
        default:
            return RATING_CLASSNAMES.good;
    }
};
```